tests/dummy/app/pods/application/template.hbs

```diff
@@ -1,9 +1,6 @@
 {{frost-select-outlet}}
-
 <div class="dummy-notification-container">
-  {{#each notifications as |notification|}}
-    {{notification-message notification=notification}}
-  {{/each}}
+{{notification-container position='top-right'}}
```

Comment:
Hmm thats not a bad idea, but right now I dont see `ember-frost-notification` being used in `core` anyway, so we can raise the issue there and open another PR to actually use it here.